alpha/scripts/utils/deleteKusersFromElastic.php

```diff
@@ -0,0 +1,122 @@
+<?php
+//require_once (dirname(__FILE__) . '/../bootstrap.php');
+require_once ('/opt/kaltura/app/alpha/scripts/bootstrap.php');
+
+const UNIX_LINE_END = "\n";
+
+if ($argc < 2)
+{
+	echo UNIX_LINE_END . ' ---- Delete Kuser From Elastic ---- ' . UNIX_LINE_END;
+	echo ' Execute: php ' . $argv[0] . ' [ /path/to/kuser_id.log || kuserId_1,kuserId_2,kuserId_3,.. ] [realrun / dryrun]' . UNIX_LINE_END;
```

Comment:
Did it meant - path/to/userIdFile ?

How will the values be in the file? one ID per line or CSV?